stock_grn/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+Installation
+============
```

Comment:
You should not use titles if not needed.

Split this with an INSTALL.rst (with just the 'To install this module ...' sentence)